www/js/history/review-controller.js

```diff
@@ -2,13 +2,50 @@ angular.module('starter.review', [])
 
 .controller('ReviewCtrl', function($scope, $stateParams) {
 
+	$scope.newReview = {};
+
+// START: Five star rating thang
+	// Initialization code
+	var fiveStarElement = document.getElementById("fiveStarRating");
+	var fourStarElement = document.getElementById("fourStarRating");
+	var threeStarElement = document.getElementById("threeStarRating");
+	var twoStarElement = document.getElementById("twoStarRating");
+	var oneStarElement = document.getElementById("oneStarRating");
```

Comment:
I'm pretty sure there's a more elegant way to go about it but given the limited timeframe I figured it'll do for now. I don't think performance is much of an issue for this prototype.